static/app/components/forms/field/fieldControl.tsx

```diff
@@ -54,7 +29,7 @@ const FieldControl = ({
   controlState,
   children,
   hideControlState,
-  flexibleControlStateSize = false,
```

Comment:
This was extranious

Reply:
hm but why? it is a boolean property with `false` as a default value 🤷‍♀️

Reply:
I mean `undefined` would produce the same result...but then it's not boolean

Reply:
A bunch of these props are like that. I don’t know why we set a default just for this one lol